DataFormats/TrackerCommon/interface/TrackerTopology.h

```diff
@@ -576,7 +576,10 @@ class TrackerTopology {
   std::string print(DetId detid) const;
 
   SiStripDetId::ModuleGeometry moduleGeometry(const DetId &id) const; 
-
+ 
+  PixelBarrelValues const& getPBVals() const { return pbVals_; };
+  PixelEndcapValues const& getPFVals() const { return pfVals_; };
```

Comment:
since you need to update in other files, please remove  semicolons at the end of the two lines above